pycurious/grid.py

```diff
@@ -212,14 +208,13 @@ def radial_spectrum(self, subgrid, taper=np.hanning, scale=0.001, **kwargs):
 
         data = subgrid
         nr, nc = data.shape
-        nw = nr
```

Comment:
The `nw` variable is the same as  the `nr`, right? In that case, there's no need to define `nw`. I've also renamed `nw` for `nr` on every occurrence in the following lines.